src/main.rs

```diff
@@ -168,12 +168,27 @@ fn handle_status<C: Config>(_matches: &ArgMatches,
 }
 
 
-subcommand!(use, add_use_subcommand {
-    about("Switch to using a different hermit shell")
+subcommand!(inhabit, add_inhabit_subcommand {
+    about("Switch to inhabiting a different hermit shell");
+    arg(Arg::with_name("SHELL_NAME").help("The name of the shell to inhabit"))
 });
 
-fn handle_use<C: Config>(_matches: &ArgMatches,
-                         _hermit: &mut Hermit<C>,
-                         _file_operations: &mut FileOperations) {
-    println!("hermit use is not implemented yet.")
+fn handle_inhabit<C: Config>(matches: &ArgMatches,
+                             hermit: &mut Hermit<C>,
+                             file_operations: &mut FileOperations) {
+    let shell_name = matches.value_of("SHELL_NAME").unwrap_or("");
+    let res = hermit.inhabit_shell(file_operations, shell_name);
+    match res {
+        Ok(v) => v,
+        Err(err) => {
+            let mes = format!("{} is a non-existant shell", shell_name);
+            println!("{}", message::error(&mes[..]));
```

Comment:
Is the slice syntax really necessary?  I think just `&mes` should be enough.  Also, this is getting kind of ugly.  It would probably be nice if we actually gave a `format`-like interface to these `message` functions eventually.  I'll make an issue for it.